etl/yaml.go

```diff
@@ -0,0 +1,45 @@
+package etl
+
+import (
+	"io/ioutil"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Transform struct {
+	Inputs []Route `yaml:"inputs"`
+	Output Route   `yaml:"output"`
+	ETLs   []Rule  `yaml:"ETLs"`
```

Comment:
I'd keep all of these lowercase.
```suggestion
	ETLs   []Rule  `yaml:"etls"`
```

Reply:
change do "transforms"